src/dmd/mars.d

```diff
@@ -1838,6 +1838,12 @@ bool parseCommandLine(const ref Strings arguments, const size_t argc, ref Param
                 error("unknown error style '%.*s', must be 'digitalmars' or 'gnu'", cast(int) style.length, style.ptr);
             }
         }
+        else if (startsWith(p + 1, "target"))
+        {
+            enum len = "-target=".length;
+            const triple = Triple(p + len);
+            target.setTriple(triple);
```

Comment:
And why not pass `p + len` directly, then make `Triple` private?

Reply:
> make Triple private

unittests